ldaputils/consts.go

```diff
@@ -13,14 +13,14 @@ package ldaputils
 
 // LDAP Key constant definitions.
 const (
-	LDAPAddressKey              = "LDAPAddress"
-	LDAPBindDNKey               = "LDAPBindDN"
-	LDAPBindPasswordKey         = "LDAPBindPassword"
-	LDAPBindSearchFilterKey     = "LDAPBindSearchFilter"
-	LDAPSubjectKey              = "LDAPSubjectKey"
-	LDAPIgnoredKeys             = "LDAPIgnoredKeys"
-	LDAPConnSecurityProtocolKey = "LDAPConnSecurityProtocol"
-	LDAPUsernameKey             = "LDAPUsername"
-	LDAPPasswordKey             = "LDAPPassword"
-	LDAPBaseDNKey               = "LDAPBaseDN"
+	LDAPAddressKey              = "address"
```

Comment:
I am not sure about the impact of this modifications

Reply:
AFAICT the direct constant is only used in Clad by the json representation: https://github.com/aporeto-inc/clad/pull/1079/files#diff-0dee619b932c87732e97d51a9d725d9bR86

Everything else uses the variable name from what I have seen.